### PR TITLE
Add Local Settings steps

### PR DESCRIPTION
**Pull request by StacyCash**
Add steps to ceate function and read local.settings.json

### CODE REVIEW

File: lessons/configuration-dotnet.md [hunk 1]
```diff
@@ -77,7 +77,69 @@ The *General settings*  section contains settings about the platform, debugging,
...
+namespace AzureFunctionsUniversity.Demo.Configuration
+{
+    public static class ReadingEnvironmentVariables
+    {
+        [FunctionName(nameof(ReadingEnvironmentVariables))]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "get", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("ReadingEnvironmentVariables Triggered via HTTP");
```
Comment (marcduiker) [inline lessons/configuration-dotnet.md:131]: I suggest to remove the logging statement. Doesn't add any value here.

**Review by marcduiker — CHANGES_REQUESTED** [review]
Just one line to remove and then I'll merge this.

File: lessons/configuration-dotnet.md (hunk 1 above)
Comment (StacyCash) [reply lessons/configuration-dotnet.md]: Removed!

**Review by marcduiker — APPROVED** [review]
LGTM, thanks!

**Pull request merged**